### DISCUSSION TITLE
快速收藏怎么设置收藏夹？

### DISCUSSION THREAD

**Discussion by dspjm** [the1812/Bilibili-Evolved#2123] [提问 / Q&A]
设置收藏到特定的收藏夹。

**Comment by the1812** [comment]
右键

**Reply by dspjm** [reply]
但是我右键点快速收藏直接是浏览器的context menu呢

**Reply by the1812** [reply]
是不是有使用什么影响右键的插件?
正常情况下右键是会显示收藏夹设定的
![image](https://user-images.githubusercontent.com/26504152/128279934-042fc3f3-0a3f-496e-a1e2-9eaf454b5710.png)
